pttrack/views.py

```diff
@@ -427,9 +444,15 @@ def byName_key(patient):
                    'title': pagetitle})
 
 
+def error_workup(request, pk):
+
+    wu = get_object_or_404(mymodels.Workup, pk=pk)
+    return render(request,
+                  'pttrack/workup_error.html',
+                  {'workup': wu})
```

Comment:
You could use a detail view for this. You just provide the template, and the detail view takes care of doing all of this for you.